src/high-level-overview.md

```diff
@@ -139,3 +139,5 @@ take:
 
 
 The first thing you may wonder if 
+
+[query model]: https://github.com/rust-lang/rust/blob/master/src/librustc/ty/maps/README.md
```

Comment:
actually, we should .. oh, maybe I didn't open that PR yet =) 

I have a PR adding a chapter for this... I'll have to remember to patch this link. :)

Reply:
I added the chapter: https://rust-lang-nursery.github.io/rustc-guide/query.html